assets/js/app.js

```diff
@@ -58,6 +66,31 @@ app.controller('AboutCtrl', ['$scope', '$rootScope', function($scope, $rootScope
   })
 }])
 
+app.controller('LoginCtrl', ['$scope', '$rootScope', function($scope, $rootScope) {
+  $rootScope.root = {
+    route: 'Login',  // this corresponds to the menu item that should be active
+    title: 'Login | GitSubmit'
+  }
+  app.controller('FormLoginCtrl', ['$scope', function($scope) {
+      $scope.submit = function() {
+        //TODO:Add code to handle the fields and send to approriate destination
```

Comment:
Same thing here, probably we should add a method showing the values (until the backend works then we'll hook this up with it).